rskj-core/src/test/java/co/rsk/vm/opcode/BasefeeDslTest.java

```diff
@@ -0,0 +1,149 @@
+package co.rsk.vm.opcode;
```

Comment:
not sure if it is needed in test classes too, but copyright header missing here.

Reply:
Totally! Will add it!